src/compiler/crystal/macros/interpreter.cr

```diff
@@ -363,9 +363,14 @@ module Crystal
         end
 
         args = node.args.map { |arg| accept arg }
+        named_args = nil
+
+        if nargs = node.named_args
+          named_args = nargs.each_with_object(Hash(String, ASTNode).new) { |arg, named_arg_hash| named_arg_hash[arg.name] = accept arg.value }
```

Comment:
```suggestion
          named_args = nargs.to_h { |arg| {arg.name, accept arg.value} }
```

Reply:
Whoa, what is this?

Reply:
https://crystal-lang.org/api/0.31.1/Enumerable.html#to_h(&block:T-%3ETuple(K,V))forallK,V-instance-method

:-)